test/integration/integration_test.go

```diff
@@ -44,15 +46,15 @@ func newDescriptorStatusLegacy(
 func TestBasicConfig(t *testing.T) {
 	t.Run("WithoutPerSecondRedis", testBasicConfig("8083", "false", "0"))
 	t.Run("WithPerSecondRedis", testBasicConfig("8085", "true", "0"))
-	t.Run("WithoutPerSecondRedisWithLocalCache", testBasicConfig("8083", "false", "1000"))
-	t.Run("WithPerSecondRedisWithLocalCache", testBasicConfig("8085", "true", "1000"))
+	t.Run("WithoutPerSecondRedisWithLocalCache", testBasicConfig("18083", "false", "1000"))
```

Comment:
Changing the port, to avoid the statstore gets reused among those tests. It is related to the go routine(https://github.com/lyft/ratelimit/blob/2c5f42aaaa74d5818829ff1473a915edda3e456c/test/integration/integration_test.go#L222-L224) won't exit after the method ends.

Reply:
OK. Can you add a TODO and comment about this? We should fix this so everything gets cleaned up after each test method but we can do that later.

Reply:
just added TODO, and also created a github issue on it.